src/api/public/apidocs-new/paths/source_project_name_package_name_binary_filename_attribute.yaml

```diff
@@ -0,0 +1,72 @@
+get:
+  summary: Get list of attributes and attribute values
```

Comment:
What about mentioning the binary/file here? This definition is maybe too generic :thinking:

Reply:
Done. It's updated